Tests/UnitTests/Core/EventData/MultiTrajectoryTests.cpp

```diff
@@ -40,77 +39,86 @@ CurvilinearParameters make_params() {
   return {cov, Vector3D(0, 0, 1), Vector3D(100, 1000, 400), -1, 0};
 }
 
-TrackState<SourceLink, CurvilinearParameters> make_rand_trackstate() {
-  return {make_params()};
-}
-
 using ParVec_t = BoundParameters::ParVector_t;
 using CovMat_t = BoundParameters::CovMatrix_t;
 
+struct ParameterContainer {
+  SourceLink sl;
```

Comment:
```suggestion
  SourceLink sourceLink;
```
For better readability.

Reply:
done